\begin{savequote}[75mm]
I think that only daring speculation can lead us further and not accumulation of facts.
\qauthor{Albert Einstein}
\end{savequote}

% pending plagiarism check
\begin{flushleft}
\chapter{Speculations on the conceptual overlap of genetic interactions and multi-view representation learning as applied to biological profiling data}

\section{Motivation}

\section{Causality}

\section{Profiling Experiments in biology}

\section{Genetic Interactions}

\section{Directional Genetic Interactions}

\section{Genetic Interaction Manifold}

\section{Multi-view representation learning}

\section{Causal representation learning}

\section{Genetic interactions as weights in a comprehensive cell state model}




Unsupervised representation learning methods, such as matrix factorization, are used to identify a lower dimensional representation of observations that consist of a large number of features (such as gene expression or morphology information). Framed differently, such representation learning methods are a form of compression algorithm, that help reduce the information required to describe biological observations. 

As stated in the introduction, multi-view representation learning is a form of representation learning that can learn factors explaining variance across multiple modalities. Put differently, multiple views of the same biology are compressed into the same low-dimensional factor space. As a practical consequence, instead of describing the biological state of an observation by listing the transcript and protein abundance of every gene, biological states can be expressed with a only few (in this case k=4) factor scores that map to both transcriptomics and proteomics data using their respective factor loadings. 

While methods like matrix factorization lead to a set of factors that can be statistically independent within the set of observations, this does not guarantee causal independence. For example, the joint distribution of observed organoid states 

P(X | T) = P(t1, t2, t3, .. , tn, p1, p2, p3, .. , pn, l1, l2, l3, .. ln | g1, g2) 


transcript
protein 
lipid

T is the distribution of treatment induced genotypes




different from work in previous chapter, we know the intervention and included its structural consequence (a change in genotype) in our joint distribution.
Given a complete exploration of the combinatorial treatment space, leading to four genotypes
In case our treatment has an effect, there will be at least one factor representing it
In case our treatment has no effect, relative to all sources of experimental noise, there will be no factor with a strong loading for the resulting change in genotype.
 
One can use this reasoning to describe the expected number of factors that should be sufficient to factorize the joint distribution of organoid states in our experiment. 

Given two different interventions, leading to 4 distinct genotypes (3 treatment induced genotypes and one reference genotype), one can deduce that k=4 factors should be the minimal and sufficient number of factors to model the joint distribution: 

P(X, T) 

can be decomposed

P(X, T) = P(f1, f2, f3, f4) 

linear genetic interaction model is framed as: 

f(g1, g2) = f(g1) + f(g2) + e

where f is the logarithm of one observed and centered feature within the set of observed features describing the phenotype

two way genetic interaction modeling 

where the number of factors corresponds to the number of variables in a linear two-way genetic interaction model

P(X, T) Ëœ g1*f1 + g2*f2 + g1*g2*f3 + f4

with f4 being the model error

Genetic interactions



\end{flushleft}