\begin{flushleft}

\chapter{Discussion}

\section{Disclosure}
Parts of this chapter have been adapted from a joint first-author publication, \textit{The drug-induced phenotypic landscape of colorectal cancer organoids} \citep{betgeDruginducedPhenotypicLandscape2022}. This discussion is divided into three sections, a general methodological section as well as discussions on findings within patient-derived organoids, and the engineered mouse colon organoids.

\section{Image-based profiling of organoids}

\subsection{Using organoids as high validity \textit{in-vitro} models for image-based profiling}

Organoids are representative \textit{in-vitro} models for diverse human tissues and can be used for image-based profiling. While the prospective use of cancer organoids as a diagnostic is currently limited by high sample dropout and long tunraround time, a high overall predictive validity for multiple therapeutic regimens has been reported \citep{ooftProspectiveExperimentalTreatment2021}. The use of organoid models in early stage drug discovery, however, is not limited by the same constraints existing in a diagnostic context. For therapeutic discovery, previous studies have successfully used organoids to perform medium-scale small molecule treatment assays. The most commonly used method is screening with an ATP based cell viability readouts \citep{vandeweteringProspectiveDerivationLiving2015}. Additionally, imaging studies with organoids have been used to characterize developmental processes such as the self-organization of intestinal cells \citep{lukoninPhenotypicLandscapeIntestinal2020, boehnkeAssayEstablishmentValidation2016a} or the morphological response to individual drugs \citep{Badder2020-au, serraSelforganizationSymmetryBreaking2019}. While image-based profiling of \textit{in-vitro} models has become an important method in functional genomics and therapeutic discovery \citep{carpenterImagebasedChemicalScreening2007}, performing such high-throughput experiments in organoid models has been a technological challenge given the 3D multi-cellular growth pattern of organoids as well as their high morphological heterogeneity. In this thesis, sparse 3D imaging as well as adjustment to the seeding and staining protocol enabled small molecule image-based profiling of patient-derived and genetically engineered colorectal cancer organoid models. 
\bigbreak


\subsection{Towards multi-view profiling}
A challenge in image-based profiling is the low interpretability of the morphological representation that is generated during the image analysis step. Classic quantitative features of cellular morphology are rarely descriptive, and even when dimensionality reduction methods or self-supervised learning techniques are used, the learnt representations are rarely directly interpretable within the context of molecular biology. To increase the interpretability of observations in a given image-based profiling experiment, supporting multi-view data, for example transcriptomic, proteomic or metabolomic data can be collected to "annotate" images. A simple and idealised approach to perform such annotation would be to collect the complete multi-view information for all tested treatment conditions (i.e. imaging, transcript abundance, protein abundance, lipid abundance). Both from a physical (some measurement methods destroy the sample) and cost perspective, however, this approach is not accessible. 
\par

In this thesis an alternative model-based approach was chose to increase the interpretability of morphology information: First, a well-annotated multi-view representation of untreated organoids was learnt by modeling morphological features as well as molecular features (i.e. transcript expression). In a second step, single-view treatment-induced morphologies were projected into the learnt representation. The resulting factor scores were used to identify treatments with an interpretable effect. Examples of such identified effects include (1) the increase in organoid size and IGF1R-signaling as a response to mTOR inhibition (factor 1, patient derived organoids); (2) the spherical reorganisation and increase in LGR5 transcript expression as a response to MEK inhibition (factor 2, patient derived organoids); as well as (3) the increase in organoid size, DNA staining intensity and canonical Wnt signaling as a response to GSK3-beta inhibition (factor 1, mouse organoids). While this approach hat utility in guiding the interpretation of otherwise challenging-to-interpret morphology information, a series of important limitations exist that are outlined below.
\par

A central limitation of the described multi-view representation learning approach are \textbf{out of distribution observations} during the projection of treated organoids. Observations during factor-learning were drawn from an untreated distribution of organoid states, while projected observations were drawn from an distinct distribution of treated organoids. As a result, an observed treatment within the query set that is far outside of the distribution learnt from the support set is likely to be misinterpreted. For example, if observation from \textit{Apc} mutant organoid lines had not been included in the support set that the multi-view representation was learnt from, the effect of GSK3-beta inhibitors on \textit{Apc} wildtype organoids would have been not interpretable. These treatment-induced morphologies would have been grouped with the set of non-specific but active treatments, such as Proteasome inhibitors. In this thesis potential out-of-distribution treatments were identified through a high AUROC score and non-specific significant treatment effects along multiple factors. In the future, more principled methods to detect out-of-distribution samples, such as obtaining the reconstruction error from projecting a sample and reconstructing it could be used to more effectively filter out-of-distribution samples. Given the limitation of out of distribution observations, it is important to include conditions in the support set that are well annotated \textbf{and} representative of the effects that are supposed to be explored, as the representation learnt from this data will be used to interpret all remaining observations within the query set. Put differently, a support set needs to be constructed with the biological question in mind, because all other observations will be interpreted using a model that was learnt from it.
\par

A second limitation associated with the used approach is a \textbf{lack of causality} in the interpretation of treatment effects on factors. For example, CDK inhibitors led to a decrese in factor 1 scores among patient derived colorectal cancer organoids, with a markedly smaller organoid size in treated organoids. This decrease in factor 1 scores does, however, not inform about the causal structure of the mechanism that is leading to a shift in factor 1. For example, while all of the following causal diagrams are as likely in the light of the factor score shift, they are distinct in terms of their biological plausibility: It is unlikely that CDK is a regulator of IGF1R signaling (CDK -> IGF1R -> organoid size), while it is more likely that CDK is a mediating variable of IGF1R signaling (IGF1R -> CDK -> organoid size) or an independent effector (IGF1R -> organoid size, CDK -> organoid size). In summary, treatment effects on factors aid in the generation of hypotheses but need to be used as starting point for further validating experiments.
\par

A third limitation of this approach is \textbf{data drift}. This limitation is most likely relevant in the context of larger profiling campaigns: Over the course of multiple experimental batches, the distribution of morphology states that is observed for both untreated and treated organoids is shifting. A model that has been trained at day 1 on a distribution of untreated organoid phenotypes and is used to interpret phenotypes on day 30 of treated organoids might show unexpected behaviour because of a shift of experimental conditions, i.e. the staining protocol. The conditions in the experiments performed during this thesis were not fully representative of such scenarios, as untreated and treated organoid morphologies were collected in parallel and all morphological data was pre-processed together (i.e. features scaling, PCA transformation). As a result the ability of a multi-view group factor analysis method to project "unseen" treatment induced morphologies into interpretable factors was assessed under idealised conditions with considerable data leakage, which needs to be accounted for if using such an approach in larger discovery or diagnostic projects.
\par
\bigbreak

Despite these limitations, the combination of multi-view representation learning and image-based profiling of organoids introduced in this thesis has the potential to be further explored and improved. A set of possible directions exist such as (1) further increasing the predictive validity of in-vitro models by imaging organoid models within coculture systems \citep{cattaneoTumorOrganoidCell2020}; (2) further increasing the robustness of multi-view profiling by introducing selective RNA-Sequencing of formalin fixed cells after a microscopy run is completed; and (3) improving the image analysis and modeling process by using self-supervised learning methods instead of classical morphological features \citep{perakisContrastiveLearningSingleCell2021}.

\section{Multi-view profiling of patient derived colorectal cancer organoids identifies factors of cancer organoid architecture}

% TODO %  introduce goal quickly
Once image-based profiling for colorectal organoids was established, a set of 11 patient derived colorectal cancer organoids were profiled in order to understand factors of molecular and morphological heterogeneity between these lines as well as their influence on small molecule treatment sensitivity differences.

\subsection{Two multi-omics factors defining patient-derived colorectal cancer organoid phenotype}

The first identified axis of phenotype variation was an IGF1R signaling program associated with increased organoid size, EGFR inhibitor resistance which can be induced by mTOR inhibition. Insulin-like growth factors are central and conserved regulators promoting cell size, organ size and organism growth \citep{pucheHumanConditionsInsulinlike2012, sunMechanismCellSize2006}. The IGF1 receptor (IGF1R) signaling cascade is activated in around 20\% of colorectal cancer patients and leads to downstream mitotic stimuli via ERK-MAPK signaling and mTOR \citep{zhongOverproductionIGF2Drives201}. In patient-derived cancer organoids, we observed that average organoid size was positively correlated with elevated IGF1R signaling activity.
\par

Increased IGF1R signaling activity also presented with a characteristic pharmacological sensitivity profile. In accordance with previous observations\citep{yaoCombinedIGF1RMEK2016}, colorectal cancer organoids in a high IGF1R signaling state were less responsive to EGFR inhibitors and more responsive to IGF1R and MEK blockade, demonstrating the central role of IGF1R mediated mitogen activated protein kinase (IGFR1-MAPK) signaling. In fact, reciprocal resistance between IGF1R and EGFR signaling inhibitors has been described in multiple cancer types \citep{huaInsulinlikeGrowthFactor2020a}. Also, organoids could be moved into a state of increased IGF1R-MAPK signaling by inhibition of mTOR, a downstream mediator of IGF1R activity. In line with this observation, a reactive induction of IGF1R signaling has been previously described as a resistance mechanism to small molecule mTOR inhibitors in cancer \citep{sharma_chromatin-mediated_2010, yoonFocalAdhesionIGF1RDependent2017a}. Its role in controlling cell size and the high number of interactions with other signaling mechanisms, such as EGFR and mTOR, have made IGF1R signaling an attractive target for therapeutic discovery. So far, however, neither mono- nor combination therapies containing IGF1R receptor inhibitors have shown clinical utility\citep{beckwithMinireviewWereIGF2015}, \citep{CostsCausesOncologya}. A speculative explanation for this failure in clinical trials might be related to its signaling network centrality. While the IGF1R receptor and its downstream effectors are are central mediators of growth-stimulating processes, they do not themselves constitute a unique dependency, which is -in contrast- often the case for genetic events involved in disease development, such as APC loss \citep{Dow2015-pc}, KRAS gain-of-function, or EGFR amplification \citep{katoRevisitingEpidermalGrowth2019}, \citep{randonEGFRAmplificationMetastatic2021}. With its high degree of connectivity within the proliferative signalling network, but no unique causal role in disease emergence, the "shock" from removing IGF1R from the signaling network through targeted inhibition might be easily compensated by neighbouring pathways such as EGFR, growth hormone, and insulin signaling \citep{beckwithMinireviewWereIGF2015}. Similar network resilience mechanisms have been observed in other networks across the life and social sciences \citep{liuNetworkResilience2022a}.
\par

While the potential of IGF1R as a target for therapeutic discovery appears mixed in light of past evidence, direct implications for the method of organoid culture - in particular the role of supplementing IGF-1 to increase organoid growth \textit{in-vitro} - might be drawn from the results of this study. Next to the observation that organoids with strong IGF1R signaling were of greater average size, the emerging role of IGF1R signaling in organoid culture was recently emphasized by the observation that addition of the IGF-1 ligand, relative to EGF, increased culture efficiency of organoids from healthy human intestinal tissue \citep{fujiiHumanIntestinalOrganoids2018a}. Given the association of IGF1R signaling with organoid size within this thesis as well as the observation from organoid isolation studies \citep{fujiiHumanIntestinalOrganoids2018a}, I hypothesize that addition of IGF-1 ligand to colorectal cancer organoid culture media could (1) further increase culture establishment efficiency and (2) reduce additional genetic bottleneck effects that might bias isolated organoid cultures towards increased IGF1R signaling.
\smallbreak

The second axis of phenotype variation was an LGR5+ program associated with cystic organoid architecture and Wnt signaling inhibitor sensitivity, which can be induced by inhibition of MEK. Organoid models with a high factor score showed a monolayer organisation with a characteristic actin cytoskeleton and an enrichment of an LGR5 intestinal stem cell signature along the factor weights. These LGR5+ organoids were more sensitive to Wnt inhibitor (Pri-724) treatment and showed a relative resistance to Erk and Mek small molecule inhibitors. Suprisingly, not only were organoids in this stem-like state more morphologically resistant to Mek inhibition, treatment with Mek inhibitors shifted organoids towards this LGR5+ state. Put differently, a subset of organoids which are LGR5+ are insensitive to MEK inhibition and Mek inhibition drives (surviving) organoids towards this LGR5+ state.
\par

The ability of ERK-MAPK inhibition (here EGFR and MEK) to move intestinal organoids into a quiescent LGR5+ state was described by \citep{basakInducedQuiescenceLgr52017c} and led to a "clutch and gas pedal" model of Wnt and ERK-MAPK signaling in which Wnt signaling is necessary and sufficient to maintain the LGR5+ stemlike state, while ERK-MAPK signaling is a primary signal to cause (stem cell) proliferation \citep{basakInducedQuiescenceLgr52017c}. Additional work by \citep{zhanMEKInhibitorsActivate2019a} recently extended this model by demonstrating that inhibition of MEK in colorectal cancer cell lines and organoid models leads to an increase in canonical Wnt signaling through inhibition of EGR1-depencent transcription of AXIN1, a necessary member of the destruction complex. According to this extended model, Wnt signaling is necessary and sufficient to maintain the LGR5+ cell state, and ERK-MAPK signaling is not only the primary proliferative signal, but its inhibition can even further shift the cellular state towards the LGR5+ identity. 


\subsection{Are patient derived organoids representative of known molecular colorectal cancer states?}

The multi-view factors that were identified in patient derived organoids during this thesis capture differences in IGF1R-, EGFR-, and canonical Wnt signaling (high Factors 1, low Factor 1, and high Factor 2, respectively). A natural question to ask is whether the identified molecular axes of variation are aligned with previously identified molecular axes or variation (and the states they define) for colorectal cancer tissue samples. Molecular states, also referred to as subtypes, of colorectal cancer have been proposed by a series of consortia and are often centered around a single data modality. Most common subtype classifications are based on bulk transcript expression data \citep{guinneyConsensusMolecularSubtypes2015} while other classifications, for example based on single-cell transcript expression \citep{joanitoSinglecellBulkTranscriptome2022}, have been developed. The Consensus Molecular Subtypes (CMS) classification \citep{guinneyConsensusMolecularSubtypes2015} is among the most widely used subtype classification and was developed by a consortium of researchers that had previously published multiple independent classifications. The CMS classification groups colorectal cancer tissue samples that have undergone bulk transcript expression measurements into four subtypes:

\begin{itemize} 
    \item CMS 1: hypermutated, BRAF-mutated, MSI and CIMP samples with corresponding signs of inflammation
    \item CMS 2: high Wnt signaling activity
    \item CMS 3: \textit{KRAS}-mutated MSS samples, signs of strong metabolic deregulation
    \item CMS 4: stromal infiltration and high TGF-beta activation, worse overall survival
\end{itemize}

Since its publication \citep{guinneyConsensusMolecularSubtypes2015}, however, the classification has not found use in clinical care. In two prospective studies in which the CMS classifier was applied and the treatment choices of Cetuximab vs. Bevacizumab (both in combination with FOLFIRI/FOLFOX) were evaluated, came to -at first sight- conflicting conclusions \citep{stintzingConsensusMolecularSubgroups2019, lenzImpactConsensusMolecular2019, aderkaExplainingUnexplainableDiscrepancies2019, sveenPredictiveModelingColorectal2019}. While one group concluded that the CMS classification had "no direct impact on clinical decision-making" \citep{stintzingConsensusMolecularSubgroups2019,} the other group concluded that their "findings highlight the possible clinical utility of CMS" \citep{lenzImpactConsensusMolecular2019}. While a range of specific explanations for this particular discordance have been proposed (i.e. cohort differences in UICC stage, pretreatment history, and chosen chemotherapy backbone), a set of general observations that highlight current limitations of clinical transcriptome measurement and classification that go beyond the use of the CMS classifier itself might be more instructive for this discussion: 

\begin{itemize} 
    \item Clinical transcriptome measurements and classification of multiple biopsies from the same patient can give discordant results, for example when comparing primary tumor and metastasis \citep{eideMetastaticHeterogeneityConsensus2021}  
    \item Clinical transcriptome measurements and classification are still not a standardised procedure, thereby reducing their reliability \citep{sveenPredictiveModelingColorectal2019}. Different transcript expression measurement methods and software libraries are used - even between clinical trials that aim to use the same classification \citep{stintzingConsensusMolecularSubgroups2019, lenzImpactConsensusMolecular2019}.
    \item Clinical transcriptome measurements capture and classify the tumor sample within its tumor microenvironment. In such contexts it is generally challenging to draw conclusions on the malignant cells in isolation.
\end{itemize}

Especially in light of the fact that the CMS classification was established on whole tissue samples, and not isolated malignant cells, it becomes clear that care should be take when comparing subtype classifications between in-vitro models and clinical samples. Within the collection of patient derived organoids profiled within this study, the majority of organoids were classified as CMS2 (11/13). Of note, no CMS1 or CMS4 organoid lines were established. This distribution of assigned subtypes was in line with reports from other colorectal cancer organoid studies \citep{vandeweteringProspectiveDerivationLiving2015, schutteMolecularDissectionColorectal2017} and showed an under-representation of CMS1 and CMS4. While CMS-subtype characteristic signatures were not recovered during the unsupervised multi-view modeling of patient derived organoids, I noticed a partial alignment of modeled states with the previously published colorectal cancer intrinsic subtype (CRIS) model \citep{isellaSelectiveAnalysisCancercell2017a}. The CRIS model was identified based on transcriptomic data from patient derived xenografts (PDX), an alternative method to propagate patient derived cancer cells ex-vivo with high efficiency. By removing genes that are not expressed in epithelial cells, the classification suppresses signal from the micro-environment and thereby reduces previously described instabilities when applying it to heterogenous clinical samples \citep{dunneCancercellIntrinsicGene2017}.
\par

Abbreviated, the five states comprising the CRIS model are: 
\begin{itemize} 
    \item CRIS A: BRAF-mutated MSI, and KRAS-mutated MSS samples
    \item CRIS B: TGF-beta pathway activation, enriched for MSI samples 
    \item CRIS C: EGFR activation, and sensitive to inhibition
    \item CRIS D: IGF signaling activation and EGFR inhibitor resistance
    \item CRIS E: high Wnt signaling activity with a "Paneth cell-like" transcriptome signature
\end{itemize}

A subset of CRIS classes was identified within the presented organoid profiling data. The gene sets defining CRIS C and D were found to be separated along factor 1, with low factor 1 scores (\textit{F1-; F2-}) associated with CRIS C and high factor 1 scores (\textit{F1+; F2-}) associated with CRIS D. There was no significant enrichment identified for the other states outlined within the CRIS model, however, a potential association between CRIS E and \textit{F2+} might motivate further investigation.
\par

While differences between tissue sample classification and organoid models are to be expected, the complete absence of CMS1, CRIS A or CRIS B organoid models is worthy of further discussion. All three subtypes show an enrichment of MSI positive colorectal cancer samples \citep{guinneyConsensusMolecularSubtypes2015, isellaSelectiveAnalysisCancercell2017a}. Throughout this project, no MSI positive organoids were isolated sucessfully. Both MSI positive colorectal cancer cell lines \citep{imkellerMetabolicBalanceColorectal2022} and PDX models \citep{isellaSelectiveAnalysisCancercell2017a} have been reported to show lower levels of intrinsic canonical Wnt signaling and an enrichment for RNF43 mutations, a Wnt signaling related functional event that -in contrast to truncating mutations of \textit{APC}- does not render cells independent of Wnt ligands \citep{vandeweteringProspectiveDerivationLiving2015}. At this point it is important to state that the quality of Wnt ligands in organoid medium is a common source of poor colon organoid culture efficiency (Kim Boonekamp, direct correspondence). Based on this observation, it is conveivable that the organoid methodology applied at the time of this study exerted a systematic bias against the development of MSI positive colon cancer organoids - possibly due to methodological challenges related to maintaining organoid culture conditions with sufficient Wnt ligand activity. Further standardising and optimising reagents related to colon cancer organoid culture will accelerate the development of fully representative \textit{in-vitro} model collections for translational colorectal cancer research. 
\par

In summary, image-based profiling of colon cancer organoids revealed characteristic differences in IGF1R-, EGFR-, and canonical Wnt signaling (high Factors 1, low Factor 1, and high Factor 2, respectively). The resulting identified organoid states showed a partial agreement with previously described molecular subtypes of colorectal cancer. At the same time, imbalances in the prevalence of certain subtypes for both CMS and CRIS within among organoid models reveals directions for further methodological improvements: both in the development of robust colorectal cancer state descriptions and in the development of low-bias organoid culture protocols. 

\section{Multi-view profiling of step-wise \textit{in-vitro} models of colorectal cancer pathogenesis}

After identifying factors of variation among patient derived organoids which represented diverse molecular states of fully developed colorectal carcinoma, the question preseted itself to what extend de-novo introduced genetic events could help understand multi-omics changes during early colon cancer pathogenesis and wether image-based profiling can be applied to discover small molecule treatment sensitivity differences in these models. 
\par

For this project murine colon organoids were chosen as a model. Murine tissue is a popular source of organoids \citep{satoSingleLgr5Stem2009} and murine organoids show a generally more robust growth \textit{in-vitro} which is an important factor to enable profiling experiments with ca. 1700 small molecule treatment conditions. An additional factor for the use of murine organoids was the availability of existing genetic mouse models, such as the \textit{LSL-Kras\textsuperscript{G12D} CreERT2} conditional allele used in this project \citep{jacksonAnalysisLungTumor2001}. While the colon, in contrast to the small intestine, is the region in which carcinomas are more frequent in humans, the inverse is true for popular genetic mouse models \citep{luoMutatedKrasAsp12Promotes2009}. Despite the mouse small intestine being more frequently transformed in genetic models of colorectal cancer, and the ease of mouse small intestinal organoid culture, colon organoid models were chosen as a model. The primary reason for this choice was to (1) more faithfully represent human colon tissue and (2) enable full control over growth and signaling factors in the medium, given that colon organoids do not contain secretory active Paneth cells. 

\subsection{Effects of \textit{Apc} truncation and \textit{Kras}\textsuperscript{G12D/+} on colon organoid models}

Apc recovery 
Wnt - characteristic sensitivity in the activity data and phenocopy by GSK * Proof of concept - recover treatments that cause a phenotype shift - most pronounced in GSK3-beta inhibitors, phenocopying due to Wnt signaling effect
Myc state:
Nucleotide synthesis - satoh 07 and cell cycle
comes with sensitivity beyond Wnt signaling inhibitors, pointing towards additinoal roles of APC
* microtubuli inhibitor, also APC direct binding
* FAK - role of APC on focal adhesions
% * mTOR role - Apc paper  - Samson (not sure wether well placed here)
\par

Kras recovery
* characteristic ERK/MAPK signaling inhibitor sensitivity:
MAPK signaling - characteristic ERK, MEK sensitivity + signs of phenotype shift in t statistic
EGFR resistance - also transcriptional downregulation seen in the gene expression data.
* Oncogene induced senescence - previously described
\par


% metabolic effects
New observations 
* Storage Lipid accumulation

lipid synthesis - reduced beta-oxidatoin, via Myc
carbohydrates - no significant effect of Warburg - as seen by Imkeller  - in fact, single cell comparison of colorectal cancer samples shows that sampels that are predominantly APC mutant are characterosed by high Myc activity *and* oxidative phosphorylation relative to tumors marked by a high rate of KRAS mutant tumors

Lipid synthesis - reduced beta oxidation
Glycolysis - up, Warburg effect, metabolic phenotype - aligned with single cell and CMS data that link KRAS presence to higher glycolysis and reduced oxidative phosphorylation 


\subsection{The interaction of Apc and Kras during colon cancer pathogenesis}
Kras and Apc interaction open question - Dow
there have beeb reports of synergy \citep{luoMutatedRasAsp122009} - our data points into a different direction, that they are mostly independent within their respective pathway 
in addition high similarity with Apc -/- models
the interaction seen is in (1) mTORC1 activtiy and inhibitor sensitivity and (2) absence of oncogene induced senescence 
increased mTOR activation rate - mtor - translation rate
increased translation rate \citep{smitDriverMutationsAdenomacarcinoma2020a} - linear

% private effects
beyond this effect - effects within their pathways
as outlined in the intro 
Apc - Wnt stemness 
Kras - growth
reminder only with Tgfb or TP53 do they show signs of invasion
\par

%limitations
limitation was the presence of Egf in the medium during image-based profiling and supporting experiments. The effect of Kras G12D might have been partially masked by the ligand in the medium. 
also high EGF dose
effects related to EGF independence by KRAS - even if only partially - are masked - 
expect to see differences in the number and size of outgrowing organoids


% perspective
\section{Perspective}

Patient derived cancer organoids are representative \textit{in-vitro} models of their tissue of origin. In therapeutic discovery, it is desireable to test a candidate against as many such high validity \textit{in-vitro} models as possible, to make confident estimates about its future clinical value. Despite the extremely high value of human \textit{in-vitro} models, the number of available models is relatively low. Today only about 500 \textit{in-vitro} models are easily accesible from large commercial-grade providers and about 2000 have been systematically characterised by international scientific consortia. 
%citation!

High validity \textit{in-vitro} models, such as organoids, are an essential component of the therapeutic discovery process. Therapeutic discovery is a sequential decision making process in which, for example, a candidate small moelcule can not, for ethical and practical reasons, be tested directly in a clinical context to observe its treatment effect. Instead, therapeutic candidates have to be evaluated using one or more models to approximate the treatment effect that could be observed in large clinical trials. When framed as an engineering problem, these in-vitro models can be considered "surrogate models", while the large clinical trial serves as an "oracle". While the term surrogate model is frequently used in engineering disciplines \citep{cozadLearningSurrogateModels2014}, it has just recently found use in the therapeutic discovery literature \citep{clydeProteinLigandDockingSurrogate2021}. In practice, multiple in-silico, \textit{in-vitro} and in-vivo surrogate models are combined and used sequentially to guide decision making during a discovery program. The higher the predictive validity and the lower the cost of using a surrogate model, the more value does it provide to the therapeutic discovery process. While determining the validity of an \textit{in-vitro} model is an empirical process, a set of axioms to prioritise models have been formulated by Vincent et al. and are presented below in a modified format \citep{vincentDevelopingPredictiveAssays2015} : 

\begin{enumerate}
    \item The model must have a clear link to the disease of interest (i.e. matched tissue of origin, representative culture conditions) and, if engineered, model the disease state based on the best understanding of the disease pathophysiology.
    \item The treatment should only represent the planned clinical intervention. Additional stimuli, such as cytokines or damage-inducing chemicals, that are required to model the disease associated phenotype should be avoided.
    \item The phenotype observed during the experiment should represent a coarse-grained function (cytotoxicity, muscle contraction) that is related to the desired clinical endpoint, rather than a molecular biomarker (transcript expression).
\end{enumerate}

Designing or "fitting" \textit{in-vitro} biological models to a disease along these axioms is a cost intensive process, but already minor changes in predictive validity of an experiment can offset these costs in a discovery program. In fact, economic research into the overall cost of therapeutic discovery by Scannel et al. \citep{scannellWhenQualityBeats2016} concludes with the statement:

\begin{quote}
"The rate of creation of valid screening and disease models may be the major constraint on R\&D productivity."
\end{quote}

% contribution - image based profiling for this model
% looking forward to using this approach 



\end{flushleft}