\begin{savequote}[75mm]
I learned that if you work hard and creatively, you can have just about anything you want, but not everything you want. Maturity is the ability to reject good alternatives in order to pursue even better ones.
\qauthor{Ray Dalio}

\end{savequote}

\begin{flushleft}

\chapter{Discussion}

My contributions to the general field of biological profiling methods described in this thesis are (1) application of the image-based profiling method to organoid models, and (2) learning biologically meaningful low dimensional representations of organoid state by pairing imaging data with additional experimental modalities through multi-view representation learning. 
\bigbreak

This discussion is divided into three sections, a general methodological part as well as discussions on findings within the patient-derived organoid project, and the engineered mouse organoid profiling project.


\section{image-based profiling of organoids}

\subsection{Organoids as high validity in-vitro models}

Organoids are representative in vitro models for diverse human tissues and can be used for image-based profiling. While the prospective use of cancer organoids as a diagnostic is currently limited by high sample dropout and tunraround time, a high overall predictive validity for multiple therapeutic regimens has been reported. The use of organoid models in early stage drug discovery, however, is not limited by the same constraints existing in a diagnostic context. For therapeutic discovery, previous studies have successfully used organoids to perform medium-scale small molecule treatment assays. The most commonly used method is screening with an ATP based cell viability readouts \cite{Van_De_Wetering2015-ko}. Additionally, imaging studies with organoids have been used to characterize developmental processes such as the self-organization of intestinal cells \cite{lukoninPhenotypicLandscapeIntestinal2020c, boehnkeAssayEstablishmentValidation2016a} or the morphological response to individual drugs \cite{badder3DImagingColorectal2020b, serraSelforganizationSymmetryBreaking2019}. While image-based profiling of in vitro models has become an important tool for the analysis of biological processes, particularly in drug discovery and functional genomics \cite{carpenterImagebasedChemicalScreening2007}, performing such high-content experiments in organoids has been a technological challenge. The primary challenges comprise the 3D growth pattern of organoid models as well as the high morphological heterogeneity. Sparse 3D imaging and projection as well as adjustment to the fragment seeding protocol presented in this thesis enabled small molecule image-based profiling of patient-derived and genetically engineered organoid disease models. 
\bigbreak

High validity in-vitro models, such as organoids, are an essential component of the therapeutic discovery process. Therapeutic discovery is a sequential decision making process in which, for example, a selected small moelcule can not, for ethical and practical reasons, be tested directly in a clinical context to observe its treatment effect. Instead, therapeutic candidates have to be evaluated using one or more models to approximate the treatment effect that could be observed in large clinical trials. In the context of optimisation theory, these models can be considered "surrogate models", while the large clinical trial being an "oracle" that is approximated. In practice, multiple in-silico, in-vitro and in-vivo surrogate models are combined and used sequentially to guide decision making during a discovery program. The higher the predictive validity and the lower the (ethical and financial) cost of using a surrogate model, the more value does it provide to the therapeutic discovery process. While determining the validity of an in-vitro model is an empirical process, a set of axioms to prioritise models have been formulated by Vincent et al. \cite{vincentDevelopingPredictiveAssays2015a} are presented below in a modified format: 

\begin{enumerate}
    \item The model must have a clear link to the disease of interest (i.e. matched tissue of origin, representative culture conditions) and, if engineered, model the disease state based on the best understanding of the disease pathophysiology
    \item The phenotype observed during the experiment should represent the desired clinical endpoint (i.e. overall tumor regression) and ideally capture a high degree of information
    \item The treatment schedule and overall experimental design should be closely aligned to assays that have been successfully used in a clinical diagnostic context
\end{enumerate}

Designing or "fitting" in-vitro biological models to a disease along these axioms is a cost intensive process, but already minor changes in predictive validity of an experiment can offset these cost in a drug discovery program. In fact, economic research into the overall cost of therapeutic discovery by Scannel et al. \cite{scannellWhenQualityBeats2016} concludes with the statement:

\begin{quote}
"The rate of creation of valid screening and disease models may be the major constraint on R\&D productivity."
\end{quote}

\subsection{Multi-view representation learning for image-based profiling}

Increasing the interpretability of the representation space within image-based profiling experiments can aid in the discovery of new biological processes. While gene-level measurements from other profiling experiments, such as transcriptome profiling, can be intuitively interpreted and used for human or algorithmic causal discovery, image-based profiling measurements suffer from a lack of mechanistic biological interpretability. For example, while changes in protein abundance after treatment with a small molecule can be directly interpreted by biologists and put into context with prior knowledge and literature to guide subsequent decision making, the same is not true for image-based profiling data. Classic quantitative features of cellular morphology are rarely descriptive, and even when dimensionality reduction methods or self-supervised learning techniques are used, the learnt representations are not interpretable within the context of our current mechanistic understanding of cellular biology. One approach to increase the interpretability of representations in a given image-based profiling experiment, is to collect a multi-view support set comprising transcriptomic, proteomic or metabolomic data to "annotate" the morphological states. This learnt factor representation can then be used to interpret observations within the remaining image set, here referred to as the query set. By choosing such an approach I enabled a higher degree of interpretability of morphological information in image-based profiling experiments, while increasing the risk of mis- or over-interpreting state changes in factor space. The underlying motivation and trade-offs involved in using multi-view representation learning for image-based profiling experiments are discussed below. 
\bigbreak

The most direct way to increase the interpretability and robustness of profiling experiments in general and image-based profiling experiments in particular would be to collect complete multi-view information for all observed conditions (i.e. imaging, transcript abundance, protein abundance, and phosphorylation state). Such experiments would also directly address the "target deconvolution problem" of phenotype-based therapeutic discovery. A treatment could be interpreted both in terms of its effect on course-grained morphological and fine-grained molecular states. Both from a physical (some measurement methods destroy the sample) and financial perspective, however, performing such profiling experiments poses a challenge. A possible solution to approximate the interpretability of such a thorough "multi-view profiling" experiment, is to divide the conditions and the corresponding observations into a thoroughly annotated small, multi-view (i.e. images, transcript abundance, protein abundance) support set and a large, uncharacterised single-view (i.e. only images) query set. The "support set" and "query set" nomenclature is taken from few shot learning - a part of the machine learning literature focused on scenarios when only few well-annotated observations are available, a common situation in biomedical research and therapeutic discovery. Once the support set is available, a simple multi-view representation is learnt from the data. It is important to include conditions in the support set that are well annotated *and* representative of the states within the phenotype space that are supposed to be explored, as the representation learnt from this data will be used to interpret the remaining observations within the query set. For example, in the mouse adenoma project, observations of all four untreated organoid models were part of the support set which then enabled describing treatments within the query set, such as GSK3b inhibitors, in terms of their ability to shift Apc wildtype organoids towards an Apc-null-like state. With a support set defined and a representation learnt, the remaining observations within the query set can then be projected into the representation space. This way hard-to-interpret but easy-to-measure morphology changes can be interpreted based on the previously learnt relationships between morphology and other, more interpretable, modalities.
\bigbreak

There are, however, risks associated with applying multi-view representation learning in profiling experiments generally, and image based profiling experiments in particular. A central limitation of the described multi-view representation learning approach are out of distribution samples - the risk of misinterpreting the state of a sample within the query set that is far outside of the distribution of states learnt from the support set. For example, if observation from Apc mutant organoid lines had not been included in the support set that the multi-view representation was learnt from, the effect of GSK3b inhibitors on Apc wildtype organoids would have been not interpretable or, even more risky, misinterpreted. Put differently, a support set needs to be constructed with care, because all other observations will be interpreted using a model that was learnt from it.
\bigbreak

A second risk associated with multi-view representation learning is overinterpretation of treatment effects that go beyond the learnt factor-level. To use the example of GSK3b inhibitors again: While causality between the treatment and the shift in factor space can be established, it is not clear what underlying mechanisms led to the shift. Without further knowledge about the small molecules, the plausible hypotheses explaining the GSK3b inhibitor's effect could include (1) inhibition of Apc, (2) activation of beta-catenin, and (3) activation of Myc based transcription, to name a few. While describing the state of the biological model using multiple factors instead of a single factor, like cell viability, reduces the degeneracy (= the number of ways a given state can be explained) of the model, there are still a multitude of possible molecular mechanisms that can lead to a shift in factors.
\bigbreak

Another challenge this approach faces is at the same time an opportunity: increasing the resolution of factors and discovering causal relationships between them. For example, factor 1 in the mouse colon organoid project was driven by (1) transcript abundance changes associated with mitosis and (2) dna repair, as well as (3) changes of cholesterol ester abundance. Factor 2, in contrast, was driven by (1) transcript abundance changes associated with cellular senescence, and (2) reduced beta-oxidation, as well as (3) accumulation of triacylglycerol lipid species. At this point, it is challenging to assign any causal hierarchy to these coarse-grained factors or the diverse processes that they contain from the data alone. With more observations being available, breaking down coarse-grained factors and establishing a directional graph between them becomes a possibility. First directed factor graph representations are already being learnt on single-view perturb-seq profiling data and show promising results in their ability to predict the phenotype of unseen treatments. Overall, identification of directed factor graphs is challenging, but new opportunities in machine learning aided causal discovery are emerging lopezLargeScaleDifferentiableCausal2022.
\bigbreak

Given the risks and limitations of interpreting learnt multi-view representations, simple, linear methods should be preferred. In the presented projects, I used multi-omics matrix factorisation (MOFA) to learn a multi-view representation. MOFA can be conceptionalised as a sparse-PCA for multi-view data and is based on the bayesian group factor analysis framework. While other methods -such as iCluster, classical group factor analysis or canonical correlation analysis- can be used to learn a representation across modalities, MOFA has been developed with design choices that make it particularly useful for biological data from profiling experiments:
\bigbreak

\begin{enumerate}
    \item feature sparsity regularisation - the model uses a spike and slab prior to reduce the number of features assigned to a given factor
    \item factor sparsity regularisation - the model includes an automatic relevance determination prior to reduce the number of factors that are active in a given view
    \item modality-specific noise models -  the model uses modality-specific prior distributions for continous, binary and count-based observations noise terms 
\end{enumerate}

Additional simplifications of MOFA include its ability to handle missing data (not supported by iCluster) and high training speed. Generally, multi-view representation learning method should perform simple transformations using modality specific features, have a tolerance to data missingness, and regularised towards sparsity. 
\bigbreak

\subsection{Perspective on organoids and  multi-view profiling experiments}
This thesis discusses the establishment of image-based profiling for patient derived and genetically engineered organoid models. Progress in image analysis \cite{chandrasekaranImagebasedProfilingDrug2021a}, the emergence of standardised protocols \cite{Bray2016-ue}, as well as cross-organisational alliances \cite{chandrasekaranJUMPCellPainting2023} are currently leading to a widespread uptake of the image-based profiling method. In the future, I expect to see a development towards in-vitro models that span tissue types using coculture methods to improve clinical predictive validity of in-vitro models. On the analytical side, I expect the multi-view representation learning approach tested in this thesis to be further explored and improved. Image-based profiling experiments are used both in academic and industry contexts and enriching learnt image representations with other modalities holds the potential to accelerate the discovery of new mechanisms and therapeutic candidates. Advances in modality-specific general purpose representation learning models, such as for biological images, will likely further increase the interest in learning and sampling multi-view representations of cellular state \cite{pfaendlerSelfsupervisedVisionTransformers2023}.







\section{Profiling of colorectal cancer organoids identifies multi-view factors of cancer organoid architecture and plasticity}
\section{Profiling of pre-malignant models of colon cancer}
\end{flushleft}