\chapter{Introduction}
\label{introduction}
\begin{flushleft}
\setlength{\parindent}{7ex}
\section{Disclosure}
Parts of this introduction, have been adapted from joint first-author publications, and were originally written by myself: \textit{Wnt signaling in cancer} \parencite{zhanWntSignalingCancer2017} and \textit{The drug-induced phenotypic landscape of colorectal cancer organoids} \parencite{betgeDruginducedPhenotypicLandscape2022}. In particular, the section "canonical Wnt signaling during colorectal cancer pathogenesis" was adapted from \parencite{zhanWntSignalingCancer2017} and the overview figure as well as its figure legend in the "thesis aims" section was taken from \parencite{betgeDruginducedPhenotypicLandscape2022}.

\section{Colorectal Cancer}
Colorectal Cancer is among the three most common and lethal forms of cancer in the developed world, accounting for ca. 10\% of all cancer incidence and mortality \parencite{sungGlobalCancerStatistics2021}. The majority of colorectal cancer cases are diagnosed after the age of 50, are located in a region spanning the rectum and sigmoid colon, and develop from macroscopic precursor lesions referred to as adenomas over the course of ca. 10 years \parencite{choGeneticAlterationsAdenoma1992}. Next to patient age, a sedentary lifestyle including obesity, diabetes, consumption of red and processed meats, alcohol, and smoking constitute risk factors \parencite{sungGlobalCancerStatistics2021}. The disease is classified by the Union for International Cancer Control (UICC) into five stages ranging from \textit{carcinoma-in-situ} (patch of malignant cells that has not yet breached the basal lamina of the intestinal mucosa), to stages 1 (malignant cells have breached the basal lamina), stage 2 (malignant cells have spread beyond the large muscle layer surrounding the colon mucosa), stage 3 (malignant cells have spread to loco-regional lymph nodes), and stage 4 (metastatic disease) \parencite{vancutsemESMOConsensusGuidelines2016a}.
\par

Among the most effective interventions to reduce the incidence and mortality rate of colorectal cancer is the removal of macroscopic lesions during preventative colonoscopy \parencite{nishiharaLongtermColorectalcancerIncidence2013}. In case the disease has further progressed, therapeutic options include oncological resection coupled with adjuvant chemotherapy (available for UICC 2, recommended for UICC 3) based on the DNA replication inhibitors 5-Fluoruracil (5FU) and Oxaliplatin (FOLFOX). Additional treatment options exist for patient subgroups such as rectal cancer patients (neoadjuvant radiochemotherapy) and frail patients (5FU instead of FOLFOX for adjuvant chemotherapy). In case of metastatic disease (UICC 4) the treatment depends on the tumor's molecular characteristics: Options for \textit{KRAS} wildtype disease include anti-Egfr antibodies (i.e. Cetuximab) in combination with chemotherapy (FOLFOX or FOLFIRI) \parencite{vancutsemESMOConsensusGuidelines2016a}. Options for \textit{BRAF} mutant disease include triple therapy (FOLFOXIRI \parencite{vancutsemESMOConsensusGuidelines2016a}), or combined Egfr-, Mek- and Braf-inhibitor treatment \parencite{kopetzEncorafenibBinimetinibCetuximab2019}. Options for tumors with microsatellite instability (MSI+) include PD-1 immune checkpoint inhibition (i.e. Pembrolizumab, anti-PD1) \parencite{andrePembrolizumabMicrosatelliteInstabilityHighAdvanced2020}. Most therapeutic options can be combined with angiogenesis inhibiting antibodies (Bevacizumab, anti-VEGFR), especially when patients are not able to receive a full treatment protocol. Additional lines of therapy can also include agents like Regorafenib and Triflouridin/Tipiracil \parencite{vancutsemESMOConsensusGuidelines2016a}. 
\par

\section{Colorectal Cancer Pathogenesis}

\subsection{The colon crypt}
The sequence of molecular events leading to colorectal cancer are governed by the tissue's stem cell biology \parencite{cleversCancerStemCell2011}. The colon stem cell niche, or crypt, is the source of all epithelial cells lining the colon. Similar to the small intestine, Lgr5+ intestinal stemcells are located at the bottom of the crypt and continuously renew the epithelium by proliferating and, through displacement, pushing newly formed cells towards the colon's lumen. This architecture serves multiple purposes, including protection of stem cells and the control of cell fate decisions across the epithelium \parencite{cleversIntestinalCryptPrototype2013a}. Multiple developmental pathways, especially Wnt signaling, Notch, Bmp and ERK-MAPK signaling, govern cell identity and growth rate in the intestinal niche \parencite{hTalesCryptNew2019}. The concentration of ligands for most of these signaling pathways is organized in gradients along the crypt-lumen axis. For example, the concentration of stem cell identity maintaining Wnt ligands (Wnt signaling) and growth-rate controlling Egf ligands (ERK-MAPK signaling), decreases as cells are pushed outside of the crypt \parencite{sasakiReg4DeepCrypt2016}. In contrast, the effect of Bmp ligands, which promote cell differentiation, increases as the concentration of BMP signaling inhibitors (i.e. Noggin) derived from basal mesenchymal cells decreases \parencite{heBMPSignalingInhibits2004}. As a cell is pushed outside the crypt by a continuous stream of newly formed, proliferating cells, the signals it receives change - leading to cellular differentiation. Similarly, if enterocyte-progenitors are moved back into the crypt, the ambient signaling leads to a de-differentiation towards an intestinal stem cell state \parencite{tettehReplacementLostLgr5Positive2016a}. \par

Given the spacial confinement of growth- and stemness-stimulating signals, the crypt architecture also leads to a protection against tissue damage. At the bottom of the crypt, a neutral competition of proliferating intestinal stem cells leads to the removal of damaged cells that show a reduced proliferation rate relative to population of surrounding stem cells \parencite{snippertIntestinalCryptHomeostasis2010a}. Given this neutral competition and the dependence on external signals to proliferate, dysfunctional or transformed cells have a higher likelihood of being removed from the niche unless they acquire a set of molecular alterations that render them independent from niche signals and increase their growth rate. \par

\subsection{The adenoma carcinoma sequence}
Colorectal cancer is a disease marked by the accumulation of genetic events in somatic cells, which over time lead to uncontrolled growth and spread beyond the tissue of origin. Both structural and functional genomics experiments have helped identify functional genetic events, so-called drivers, that cause the transition from controlled to malignant cellular state. A classic model for the sequence of functional events leading to colorectal cancer is known as the adenoma-carcinoma sequence \parencite{vogelsteinGeneticAlterationsColorectaltumor1988}. It starts with the activation of the Wnt signaling pathway through loss of the tumor suppressor \textit{APC} and is followed by the activation of the ERK-MAPK signaling pathway through mutations of the \textit{KRAS} or \textit{BRAF} oncogene. Further mutations activating PI3K signaling (i.e. \textit{PI3K} oncogene, \textit{PTEN} tumor suppressor) and inhibiting Tgf-beta (i.e. \textit{SMAD4} tumor suppressor) and p53-signaling (i.e. TP53 tumor suppressor) complete the transformation from adenoma to invasive carcinoma \parencite{fearonMolecularGeneticsColorectal2011}. \par

When simplified, colorectal cancer forms along the adenoma-carcinoma sequence through (I) a common chromosomal instability (ca 80\%) or (II) a DNA-mismatch repair deficiency (ca. 20\%) associated mechanism \parencite{markowitzMolecularOriginsCancer2009} and \parencite{pancioneGeneticEpigeneticEvents2012}. These two trajectories of tumor development have been associated with characteristic clinical, pathological, and molecular properties. 
\par

Tumors of the common chromosomal instability phenotype present themselves early as common, non-serrated polyps in the rectum and sigmoid colon \parencite{markowitzMolecularOriginsCancer2009}. Most tumors are microsatellite stable and have frequent \textit{APC} truncating mutations coupled with \textit{KRAS} activating mutations. 
\par 

In contrast, tumors of the DNA-mismatch repair phenotype are frequently present as serrated polyps and more likely to be located in the right colon \parencite{markowitzMolecularOriginsCancer2009}. These tumors are often marked by a CPG-island methylation phenotype (CIMP) that can lead to the hypermethylation of tumor suppressors and DNA-mismatch repair genes \parencite{oginoCpGIslandMethylator2009}. Consequently, these tumors have a higher proportion of microsatellite instability, higher mutational burden, higher immunogenicity, and thus a higher immune-cell infiltration. APC mutations are less frequent and if present more likely to be missense than truncating \parencite{borowskyRoleAPCWNT2018}. Mutations of the BRAF oncogene are more frequently found than \textit{KRAS} mutations. 
\par

Independent of the mechanism, the sequential accumulation of functional genetic events renders colorectal cancer cells progressively independent of the organism's tissue control mechanisms and thus enable the uncontrolled, independent growth of cells which defines late stage cancer.

\subsection{The role of APC and KRAS during colorectal cancer pathogenesis}
Genetic alterations in the tumor supressor \textit{APC} and the proto-oncogene \textit{KRAS} are among the two most frequent mutations found in across colorectal cancer cases \parencite{markowitzMolecularOriginsCancer2009}. Alterations in the two genes are highly correlated and found already at early stages \parencite{minaConditionalSelectionGenomic2017}. What defines the interaction between the two colorectal cancer drivers, is, however, still an open question \parencite{parsonsWNTDriverDependency2021}. In the following section the mechanism of Wnt signaling and ERK-MAPK signaling are briefly introduced, as these two pathways are directly affected by Apc loss and Kras activation, respectively, and both have a regulatory function within the colon stem cell niche \parencite{hTalesCryptNew2019}.

\begin{figure}[h]
\centering
\includegraphics[width=0.55\textwidth,
                keepaspectratio]{figures/adenomaprofiling/pdf/fig_0_1.pdf}
\caption[Simplified illustration of ERK-MAPK signaling and canonical Wnt signaling]{\textbf{Simplified illustration of Egf dependent ERK-MAPK signaling and canonical Wnt signaling a} Egf dependent signaling cascade. Ras is highlighted in blue. \textbf{b} canonical Wnt signaling cascade. The destruction complex, including Apc, is highlighted in blue.}
\label{fig_180}
\end{figure}
\bigbreak

\subsubsection{canonical Wnt signaling during colorectal cancer pathogenesis}

Members of the Wnt signaling pathway were first identified as regulators of embryonic development \parencite{sharmaEffectWinglessWg11976, nusslein-volhardMutationsAffectingSegment1980} and are controlling adult stem cell identity across tissues. As a result, members of the pathway are frequently altered during cancer initiation \parencite{zhanWntSignalingCancer2017}. In canonical Wnt signaling, absence of Wnt ligands leads to phosphorylation of beta-catenin by the destruction complex, which contains the scaffold protein Axin, the large disordered protein Apc (Adenomatous polyposis coli, \textit{APC}) and the kinases Gsk3-beta as well as Casein kinase \parencite{zhanWntSignalingCancer2017} (Figure \ref{fig_180} b). 
In this state, beta-catenin is phosphorylated by Gsk3-beta, ubiquitinated by beta-TrCP and subsequently targeted for proteasomal degradation. In the absence of nuclear beta-catenin, the transcriptional repressive complex containing TCF/LEF and transducing-like enhancer protein (TLE/Groucho) recruits histone deacetylases to repress target genes. \par 

The role of Wnt signaling during colorectal cancer development is well established \parencite{polakisManyWaysWnt2007}. Here, loss of \textit{APC} is the most frequent driver of Wnt signaling in colorectal cancer and can be found in about 80\% of colorectal cancer patients \parencite{fearonMolecularGeneticsColorectal2011, zhanWntSignalingCancer2017}. Both genetic mouse models and \textit{in-vitro} experiments with human colon organoids (introduced below) have demonstrated that loss of function of \textit{APC} (\textit{Apc} in mice) is sufficient to cause adenoma formation \parencite{moserApcMinMouseModel1995}, which is marked by Wnt-ligand independent proliferation \parencite{matanoModelingColorectalCancer2015a, drostSequentialCancerMutations2015}. Further, in the context of additional functional events in \textit{Kras} and \textit{Trp53}, \textit{Apc} loss is necessary to sustain invasive tumor growth of \textit{in-vivo} cancer models \parencite{dowApcRestorationPromotes2015a, sakaiCombinedMutationApc2018a}.
\par

\subsubsection{ERK-MAPK signaling during colorectal cancer pathogenesis}

The extracellular-signal-regulated (ERK) mitogen-activated protein kinase family (MAPK) is one of three major MAPK families, together with the JNK (c-jun N-terminal kinase or stress-activated protein kinases) and MAPK14 group of protein kinases \parencite{zhangMAPKSignalPathways2002}. These kinases and the signaling cascades they form play a major role in (I) integrating external proliferative signals, (II) reacting to stress or ambient cytokines and (III) protecting cells from apoptosis, respectively \parencite{fangMAPKSignallingPathways2005}. The ERK-MAPK signaling cascade itself is constituted by a set of protein kinase subfamilies, Ras (encoded by i.e. \textit{KRAS}, \textit{NRAS}, \textit{HRAS}), Raf (encoded by \textit{BRAF}, \textit{CRAF}, \textit{ARAF}), Mek (encoded by \textit{MEK1}, \textit{MEK2}) and Erk (\textit{ERK1}, \textit{ERK2}) which transmit extracellular mitogenic stimuli in a step-wise phosphorylation-based signaling cascade towards downstream transcription factors (Figure \ref{fig_180} a). \par

In the colon crypt, the ERK-MAPK signaling cascade and its members are key regulators of cell proliferation and tissue growth rate \parencite{hTalesCryptNew2019}. Blockade or removal of Egf, a canonical ligand of the signaling cascade's upstream receptor (encoded by \textit{EGFR}), leads to a cell cycle arrest of proliferating Lgr5+ cells while not affecting their cellular state \parencite{basakInducedQuiescenceLgr52017}. As a result, Lgr5+ cells can re-enter proliferation as soon as ERK-MAPK signaling is restored.
\par

In colorectal cancer, the Ras kinase subfamily members are mutated in about 36\% of cases \parencite{fangMAPKSignallingPathways2005}. Next to Ras, Raf mutations can also be found in around 10\% of colorectal cancers \parencite{fangMAPKSignallingPathways2005}. Of note, mutations of \textit{KRAS} and \textit{BRAF} occur mostly in a mutually exclusive pattern (introduced above) \parencite{fangMAPKSignallingPathways2005, minaConditionalSelectionGenomic2017}. According to the Vogelstein model of colorectal cancer initiation \parencite{fearonMolecularGeneticsColorectal2011}, activating mutations of \textit{KRAS} takes place early during cancer development, more specifically, after loss of \textit{APC}. \textit{in-vivo} mouse models of the oncogenic \textit{Kras}\textsuperscript{G12D/+} show that this functional event is not sufficient to cause adenoma formation in isolation, but instead leads to \textit{Cdkn2a}-dependent oncogene-induced senescence \parencite{benneckeInk4aArfOncogeneinduced2010}. In the context of \textit{in-vivo} \textit{Apc} loss of function (\textit{Apc}\textsuperscript{$\Delta716$}), however, the \textit{Kras}\textsuperscript{G12D/+} genotype leads to an increased number of adenomas and reduced survival \parencite{sakaiCombinedMutationApc2018a}. Of note, the combination of \textit{Apc}\textsuperscript{$\Delta716$} and \textit{Kras}\textsuperscript{G12D/+} in colon epithelial cells are, however, not sufficient to cause significant invasive and metastatic disease \textit{in-vivo} \parencite{sakaiCombinedMutationApc2018a, boutinOncogenicKrasDrives2017a}, which is only seen in triple mutant genotypes (i.e. \textit{Apc}\textsuperscript{$\Delta716$} / \textit{Kras}\textsuperscript{G12D/+} / \textit{Tgfbr2}\textsuperscript{-/-}; or \textit{Apc}\textsuperscript{-/-} / \textit{Kras}\textsuperscript{G12D/+} / \textit{Trp53}\textsuperscript{-/-}). Here, in the context of an \textit{Apc}\textsuperscript{-/-} and \textit{Trp53}\textsuperscript{-/-} genotype, \textit{Kras}\textsuperscript{G12D/+} is necessary and sufficient to lead to invasive and mestatic disease \textit{in-vivo}. Conversely, removal of the \textit{Kras}\textsuperscript{G12D/+} in fully formed tumors \textit{in-vivo} leads to a regression back towards an adenoma phenotype \parencite{boutinOncogenicKrasDrives2017a}.



\section{Organoids}

Organoids are three-dimensional cell culture models from primary adult tissue. Intestinal organoids develop from Lgr5+ adult stem cells and were first isolated from the small intestine of mice \parencite{satoSingleLgr5Stem2009}. Subsequently, further organoid models across tissue-types and species have been developed. These include organoids from healthy and cancerous human large intestine \parencite{satoLongtermExpansionEpithelial2011}, pancreas \parencite{driehuisPancreaticCancerOrganoids2019}, mammary epithelium \parencite{zhangEstablishingEstrogenresponsiveMouse2017, sachsLivingBiobankBreast2018} and the hepatobiliary system \parencite{huchVitroExpansionSingle2013}. The number of tissues that can be modeled through organoids has been continuously increasing since the first publication of this methodology.

\subsection{Isolation and Culture of Organoids}

Culturing organoids from primary cells requires the addition of specific tissue-dependent growth factors and the embedding of cells in 3D hydrogels \parencite{merkerGastrointestinalOrganoidsHow2016}. In the case of colon organoids, the necessary growth factors are inspired by signaling cues available in the intestinal stem cell niche (discussed above): Wnt and R-spondin ligands to activate and maintain canonical Wnt signaling; EGF ligands to stimulate ERK-MAPK signaling, and Noggin ligands to inhibit the differentiating effects of the BMP signaling cascade \parencite{satoGrowingSelforganizingMiniguts2013}. When combined with inhibitors of TGFbeta and p38 mediated signaling, these growth factors enable organoid formation and continous proliferatation ex-vivo. In line with the niche-focused model of colon cancer formation outlined previously, patient derived colorectal cancer organoids proliferate in simpler conditions in a niche-independent manner that corresponds to their genetic state (i.e. \textit{APC} mutant colon organoids grow in conditions lacking Wnt and R-spondin ligands) \parencite{Fujii2016-ax}. Not only the high isolation efficiency of up to 80\%, but also the high genetic and transcriptomic correlation with their tissue of origin have made organoids an increasingly popular translational research model \parencite{pauliPersonalizedVitroVivo2017a}. Both the high isolation efficiency, and the molecular representation of the tissue of origin are likely a result of the culture conditions which are emulating the tissue-specific stem cell environment. \par

\subsection{Diagnostic use of Colorectal Cancer Organoids}

Given the benefits of the methodology, patient derived organoids are being trialed as predictive models for personalised treatment recommendation in colorectal cancer care \parencite{vandeweteringProspectiveDerivationLiving2015, vlachogiannisPatientderivedOrganoidsModel2018, ganeshRectalCancerOrganoid2019, ooftPatientderivedOrganoidsCan2019, yaoPatientDerivedOrganoidsPredict2020a}. In these assays, personalised organoid models are generated and then treated with therapeutic options \textit{in-vitro}. While robust reports on the predictive validity of organoids for progression-free survival are still missing, current results for response rate models show noteworthy predictive validity for Irinotecan, FOLFIRI, and neoadjuvant radiation therapy. A pooled analyses of the previously published literature recently summarised the aggregate sensitivity and specificity for organoid based treatment response rate prediction at 0.81 and 0.74, respectively (conservative single-point AUROC estimate: 0.78) \parencite{wensinkPatientderivedOrganoidsPredictive2021, zhangNoteROCAnalysis2005}. Despite the potential of patient derived organoids to guide clinical decision making, the method is not ready for use in a clinical context given a high biopsy-to-evaluation dropout rate and a long turnaround time being the primary technical hurdles. A recent failed single-arm prospective clinical trial for organoid-based last-line treatment recommendations reported only 6 treated patients out of 61 patients that were initially included in the study (>90 \% dropout rate, 23 participants lost due to organoid establishment failure, 11 participants lost due to disease progression under standard-of-care before a recommendation could be made) \parencite{ooftProspectiveExperimentalTreatment2021}. Further improvements of reagents, isolation procedures, and \textit{in-vitro} protocols might lead to lower dropout rates and turnaround times to enable further investigation of organoid-based diagnostics for personalised colorectal cancer care. \par

\subsection{Therapeutic Discovery using Colorectal Cancer Organoids} 

Next to diagnostics, organoid have been evaluated as models for therapeutic discovery. Patient derived organoids can be processed in high-throughput small molecule screens with, ATP-based, cell viability readouts \parencite{vandeweteringProspectiveDerivationLiving2015, boehnkeAssayEstablishmentValidation2016a}. In addition, organoids are amenable to efficient genetic engineering using CRISPR \parencite{matanoModelingColorectalCancer2015a, drostSequentialCancerMutations2015}. This opens up the possibility of evaluating the effect of therapeutic candidates against precisely defined, engineered genetic disease states. Next to cell viability measurements, readouts such as systematic microscopy have been used to study the self-organization of healthy intestinal organoids \parencite{lukoninPhenotypicLandscapeIntestinal2020}. However, a the large morphological heterogeneity of organoids as well as their three-dimensional, multi-cellular architecture present a challenge to these methods.

\section{Profiling Experiments} 

Profiling experiments are biological experiments in which high dimensional phenotypes are observed of \textit{in-vitro} models that are subjected to multiple treatments. Examples for such methods include Transcriptome Profiling Experiments (i.e. Perturb-Seq \parencite{dixitPerturbSeqDissectingMolecular2016}, L1000 Profiling \parencite{subramanianNextGenerationConnectivity2017}) as well as Image-based Profiling Experiments \parencite{caicedoApplicationsImagebasedProfiling2016} (described below). These experiments generate data containing (1) multiple features (i.e. transcript count or morphological feature) for (2) multiple treatments (i.e. small molecules or CRISPR perturbations) and (3) multiple experimental units (i.e. multiple genetically defined cell states, multiple organoids). Of note, the presence of high dimensional features for multiple treated states is what separates profiling experiments from observational (i.e. many features, such as single-cell RNA-Seq, across multiple clinical samples, but no prospective treatment) or focused screening projects (i.e. one feature, such as cell viability, across multiple treatments). 

\subsection{Image-based Profiling}

Image-based profiling is an unbiased high-throughput microscopy based profiling method in which \textit{in-vitro} models are systematically treated with small molecules or genetic perturbations (i.e. siRNA) to observe their treatment-induced phenotype \parencite{carpenterImagebasedChemicalScreening2007, caicedoApplicationsImagebasedProfiling2016}. A hallmark of image-based profiling is the emphasis on capturing unbiased morphological information from treated cells. Different from ATP-based cell viability readouts (where every well on a plate corresponds to one scalar measurement), every experimental unit (i.e. individual cell, organoid) is detected and multiple descriptive features (i.e. size, staining intensity, texture) are measured. As a result, every treatment condition (well on a plate) corresponds to a table with multiple biological units as rows and multiple features as columns, and thereby enables the ability to investigate biological heterogeneity within a given treatment condition. In contrast to screening experiments which are focused on one particular phenotype of interest, image-based profiling experiments are using general purpose staining protocols (i.e. DNA, Actin, Mitochondria, and the cellular membrane). Given the large amount of collected information from such experiments, analysing their results is a common challenge and subject of continuous method development \parencite{chandrasekaranImagebasedProfilingDrug2021}. 
\par

The image-based profiling method is commonly used in functional genomics \parencite{lauferMappingGeneticInteractions2013, billmannGeneticInteractionMap2016} and translational biomedical research \parencite{gibsonStrategyIdentifyingRepurposed2015, moshkovPredictingCompoundActivity2023}. While the exact approaches differ, a set of common steps in image-based profiling analysis exist: (1) Image Analysis, (2) Phenotype Modeling and (3) Novelty Detection. During Image Analysis, raw microscopy images are processed using manual or learnt feature extraction methods to generate a low-dimensional representation of each treatment-induced phenotype. Ideally, such representations are comparable across contexts, captures biological differences, and suppresses technical artefacts. Examples for different methods include classic texture feature calculations followed by Principal Component Analysis (PCA) \parencite{caicedoDataanalysisStrategiesImagebased2017} as well as modern approaches such as end-to-end self-supervised learning with a contrastive-loss \parencite{perakisContrastiveLearningSingleCell2021}. During Modeling, a set of conditions that are well annotated are observed and the low-dimensional representations of their treatment-induced phenotypes are modeled as a function of these treatment conditions. For example, the phenotype of siRNA treated Drosophila cells has been modeled as an effect of two combined siRNAs using a linear model \parencite{billmannGeneticInteractionMap2016}, and the phenotype of small-molecule treated human cells has been modeled as an effect of a single disease allele using a boosting-based model \parencite{gibsonStrategyIdentifyingRepurposed2015}. During Novelty Detection, a set of treatment conditions of interest are identified based on their deviations from the learnt model. For example, the phenotype of Drosophila cells might change more drastically than anticipated under a linear model after treatment with two siRNAs which target genes in the same signaling pathway (indicating a form of epistasis) \parencite{billmannGeneticInteractionMap2016}. Analogously, a small-molecule induced phenotype of human cells might be misclassified as wildtype, although the treated cells are bearing a disease allele (indicating a potential small molecule candidate inhibitor of the disease associated molecular mechanism) \parencite{gibsonStrategyIdentifyingRepurposed2015}. Once identified, these novelties can be further validated in independent experiments to identify potential new biological mechanisms or therapeutic candidates.
\par

\subsection{Matrix Factorization}
The natural representation of data generated by high-throughput experiments, including profiling experiments, are matrices containing the measured treatment condition (rows) and their observed features (columns). A common challenge in large biological data analysis is the identification of interpretable biological processes that underlie the observed data \parencite{stein-obrienEnterMatrixFactorization2018}. The basic goal of matrix factorization methods is to identify a small number of underlying factors (i.e. biological processes) within high-dimensional data. 

\begin{equation}
    Y = ZW^T + \epsilon
\end{equation}

In this process, a data matrix \( Y \), consisting of \( n \) samples and \( m \) features, is decomposed into two lower-dimensional matrices: 

\begin{enumerate}
    \item The factor score matrix \( Z \), which has dimensions \( n \times k \), where \( k \) represents the number of latent factors. Each row in \( Z \) corresponds to a sample, and each column corresponds to a latent factor.
    \item The factor weight matrix \( W^T \), which has dimensions \( k \times m \). Here, each row corresponds to a latent factor, and each column corresponds to a feature. This matrix is also referred to as the loading matrix. 
\end{enumerate}

The factor score matrix \( Z \) can be understood as a learnt, lower-dimensional representation of the data with \( k \) dimensions. The choice of \( k \) (the number of latent factors) ideally strikes a balance between simplicity and comprehensiveness and can be algorithmically determined. 

Different matrix factorization methods exist, with Principal Component Analysis (PCA), non-negative matrix factorization (NMF) and Independent Component Analysis (ICA) being among the most common \parencite{stein-obrienEnterMatrixFactorization2018}. Each of these methods performs the basic decomposition outlined above, but apply different constraints during the decomposition \parencite{stein-obrienEnterMatrixFactorization2018}: In PCA the identified factors are orthogonal to each other and are ranked by the amount of variance they explain within the data. In NMF both the factor score and weight matrix must be non-negative, which aides interpretation. In ICA the factors are statistical independent and non-gaussian, which is useful for the analysis of time-course data, such as audio or electrical activity. Each of these three popular methods do not include further assumptions, such as a dedicated measurement error \(\epsilon\).

In case the data generating process is better understood, additional assumptions can be introduced. For example, directly modeling the measurement error for each feature \(\epsilon\) and applying sparsity constraints to the matrix factorization are options. If such additional assumptions are made, the method is more commonly referred to as a form of factor analysis \parencite{klamiGroupFactorAnalysis2014}.


\subsection{Extending Matrix Factorization to multiple experimental Views}

Many physical processes of scientific interest can be observed through more than one method or "view". For example, a patient tissue sample can be characterised through DNA sequencing, RNA sequencing and lipid abundance measurements. Finding a small number of underlying factors across high-dimensional data coming from multiple views is thus an increasingly frequent problem in biological data analysis and is the focus of the multi-view representation learning field \parencite{liSurveyMultiViewRepresentation2019}. In group factor analysis methods \parencite{virtanenBayesianGroupFactor2012, klamiGroupFactorAnalysis2014}, which include multi-omics factor analysis (MOFA, introduced further below) \parencite{argelaguetMultiOmicsFactorAnalysis2018b, argelaguetMOFAStatisticalFramework2020a}, the factor analysis described above is extended to multiple data views (also referred to as modalities): A set of \(m\) data matrices \( Y_m \) is decomposed into a single factor score matrix \( Z \) and a corresponding set of \(m\) weight matrices \( W_m^T \) and error terms \(\epsilon_m\). 

\begin{equation}
    Y_m = ZW_m^T + \epsilon_m
\end{equation}

Factors that are identified using this approach describe patterns within the data that can span multiple different data modalities. Next to simplifying the interpretation of complex data, factors that have been learnt to represent the structure of measurements across multiple modalities with group factor analysis have been shown to enable more reliable similarity estimation and missing data prediction than alternative factor analysis methods or simply raw features taken from the measured data \parencite{klamiGroupFactorAnalysis2014}.
\par

Multi-omics matrix factorisation (MOFA) builds on group factor analysis to learn a multi-view representation from data generated by omics-methods \parencite{argelaguetMultiOmicsFactorAnalysis2018b, argelaguetMOFAStatisticalFramework2020a}. While other methods, such as classical group factor analysis, can be used to learn a representation across multiple modalities, MOFA has been developed with design choices that make it particularly useful for complex biological data:

\begin{enumerate}
    \item feature sparsity - the model reduces the number of features assigned to a given factor
    \item factor sparsity - the model reduces the number of factors that are assigned to a given modality
    \item modality-specific priors - the model uses dedicated priors for continuous, binary and count-based modalities 
    \item missing data handling - the model can be trained on incomplete observations
    \item speed - the model is fast to run compared to alternative methods, such as group factor analysis
\end{enumerate}
 
\bigbreak
In the context of the work presented in this thesis, multi-omics factor analysis (MOFA) was used to learn multi-modality representations for both patient derived and genetically engineered colon organoids. The modeled data comprised modalities such as organoid morphology, transcript expression, protein-, and lipid- abundance. The identified factors corresponded to major axes of biological variation and were used to interpret results from image-based profiling experiments as outlined below. 

\subsection{Using Matrix Factorization to interpret observations}

While matrix factorization methods are commonly used to learn the intrinsic structure of high-dimensional data within an exploratory context, it can also be used to guide decision making by predicting similarities between observations. In the latter context, a well-annotated subset of the data, here referred to as the support set, is first used to learn factors. In a second step, another set of observations which are to be investigated, here referred to as query set, are projected into the learnt representation to estimate their properties. The "support set" and "query set" terminology is adapted from the field of meta-learning - a part of the machine learning literature focused on scenarios when only few well-annotated observations are available \parencite{hospedalesMetaLearningNeuralNetworks2020}, a common situation in biomedical research and therapeutic discovery. 
\par

In order to estimate the factor scores for new observations in the query set the previously identified factor weight matrix is used. 

\begin{equation}
    Z_{\text{new}} = Y_{\text{new}} W^+
\end{equation}

Instead of decomposing the query data matrix \(Y_{\text{new}}\) into a factor score matrix and factor weight matrix, as done during the initial learning, the query data matrix  \(Y_{\text{new}}\) is multiplied with an approximated inverse of the factor weight matrix \( W^+ \) to estimate the corresponding factor scores \( Z_{\text{new}} \).
\par

In practice, scenarios in which such a multi-step approach is applied include for example recommender systems \parencite{korenMatrixFactorizationTechniques2009}. Here, a new observation is made and put in context with existing observations to estimate its properties and guide decision making (i.e. which experiment to run, or a what product to recommend). In this thesis a related approach is used for multi-view profiling of \textit{in-vitro} organoid models. This approach is useful especially when cost differences between data modalities exist and treatment conditions with an effect of interest are rare: Instead of observing and modeling all possible treatments with multiple views at once (i.e. transcript expression, morphology, protein abundance), a representative subset, the support set, is first selected and fully observed to learn a multi-view model. This subset includes well annotated conditions, such as untreated \textit{in-vitro} models of healthy and diseased tissue. In a second step, a large number of treatment conditions, the query set, are observed with only one modality. This modality is affordable and sufficiently informative (i.e. microscopy images). In the third and final step, the single-view observations are projected using the previously defined multi-view model. Provided the support set was representative enough, treatment conditions that show an effect of interest (i.e. a phenotype deviates from the expectation) are identified as novel and can be further characterised by making observations from additional views.

\newpage
\section{Aims of Thesis}
Organoids are high fidelity three-dimensional \textit{in-vitro} models that can be generated from cancer and healthy tissue. Genetically engineered organoid models demonstrate strong morphological differences. Similarly, patient derived organoids show pronounced morphological heterogeneity between donors, as well as within donors \textit{in-vitro}. In both cases, the factors determining this heterogeneity are not well understood. In addition, it is not clear whether these morphological differences are associated with differences in small molecule treatment sensitivity \textit{in-vitro} and whether morphological states can be manipulated through small molecule treatment.
\bigbreak

The aim of this thesis is to understand sources of morphological variation in colorectal cancer organoid models. To this end, organoid models were generated and image-based profiling was performed to identify multi-view factors explaining both morphological and molecular heterogeneity (Figure \ref{fig_130}). In addition, the relationship between factors and small molecule treatment sensitivity was investigated. Here, besides measuring relative differences in treatment sensitivity, the aim was to also identify non-lethal treatments that changed organoid molecular and morphological state along these newly identified axes of variation.
\bigbreak

\begin{figure}[h]
\centering
\includegraphics[width=\textwidth,
                height=\textheight,
                keepaspectratio]{figures/promise/pdf/fig_1_1_1.pdf}
\caption[Overview of organoid multi-view profiling experiments]{\textbf{Overview of organoid multi-view profiling experiments} Organoids were a) isolated from endoscopic biopsies from patients with colorectal cancer or b) derived from mouse tissue and modified using CRISPR and conditional allele expression. Organoids were dissociated and evenly seeded in 384-well plates before perturbation with small molecules (ca. 800 and 1700 conditions for patient derived and mouse organoids, respectively). After treatment, high-throughput fluorescence microscopy was used to capture the morphology of organoids.  The multi-channel (DNA, beta-actin, cell permeability) 3D imaging data was projected, segmented, and descriptive features were extracted to quantify potential drug-induced phenotypes. Untreated organoid morphology, organoid size and drug activity scores were integrated with additional modalities, such as mRNA expression and DNA sequencing data in a Multi-Omics Factor Analysis (MOFA) to increase interpretability of organoid variation. Figure adapted from \textit{The drug-induced phenotypic landscape of colorectal cancer organoids} \parencite{betgeDruginducedPhenotypicLandscape2022}}
\label{fig_130}
\end{figure}

To understand sources of heterogenity for both models of fully developed and precursor states of colorectal cancer, the thesis is separated into two parts. In the first part, 11 patient derived colorectal cancer organoids were isolated and image-based profiling was performed. In the second part healthy colon organoids were genetically engineered to generate models of wildtype, \textit{Apc}\textsuperscript{-/-}, \textit{Kras}\textsuperscript{G12D/+}, and \textit{Apc}\textsuperscript{-/-} / \textit{Kras}\textsuperscript{G12D/+} genotype and profiled in a similar fashion. In both cases multi-view factors and their characteristic treatment sensitivities were identified and small molecules with non-toxic, factor-specific effects were investigated.

To summarise, the three central questions this thesis aims to answer are:
\begin{enumerate}
    \item What are the molecular factors determining organoid morphology in colorectal cancer models?
    \item Are these axes of variation associated with differences in small molecule treatment sensitivity? 
    \item Can small molecules shift organoids in their molecular state along these factors? If so, what are the mechanisms through which such small molecules exert their effect?
\end{enumerate}

These questions were addressed by (1) establishing image-based profiling for colorectal cancer organoid models, (2) multi-view modeling of organoid morphology together with their molecular state, and (3) using the resulting multi-view factors to interpret differences as well as to identify small molecule treatments that caused molecular state changes along them.

\end{flushleft}