\begin{savequote}[75mm]
Details matter, it’s worth waiting to get it right.
\qauthor{Steve Jobs}
\end{savequote}

% not yet modified
\begin{flushleft}
\chapter{Materials and Methods}

\section{Patients}
All patients were recruited at the University Hospital Mannheim, Mannheim, Germany. We included untreated patients with a new diagnosis of colon or rectal cancer in this study and obtained biopsies from their primary tumors and adjacent normal tissue via forceps based endoscopy. Exclusion criteria were active HIV, HBV or HCV infections. Biopsies were transported in phosphate buffered saline (PBS) on ice for subsequent organoid extraction. Clinical data, tumor characteristics and molecular tumor data were pseudonymized and systematically collected in a manual prospective database. The study was approved by the Medical Ethics Committee II of the Medical Faculty Mannheim, Heidelberg University (Reference no. 2014-633N-MA and 2016-607N-MA). All patients gave written informed consent before tumor biopsy was performed. In this study, we extracted PDOs from 25 patients with colorectal cancer, 10 of them female, 15 male, with a mean age of 66 years (median 65). 16 patients had a rectum carcinoma, 9 a colon carcinoma. Detailed patient characteristics, including stage and treatment data can be found in Table S1.

\section{Organoid Culture}

\subsection{Patient Derived Organoid Culture}
Patient derived organiod (PDO) cultures were extracted from biopsies as reported by Sato et al. \cite{Sato2011} with slight modifications. Tissue fragments were washed in DPBS (Life technologies) and digested with Liberase TH (Roche) before embedding into Matrigel (Corning) or BME R1 (Trevigen). The medium, termed ENA, contained Advanced DMEM/F12 (Life technologies) medium with Pen/Strep, Glutamax and HEPES (basal medium) supplemented with 100 ng/ml Noggin (Peprotech), B27 (Life technologies), 1,25 mM n-Acetyl Cysteine (Sigma), 10 mM Nicotinamide (Sigma), 50 ng/ml human EGF (Peprotech), 10 nM Gastrin (Peprotech), 500 nM A83-01 (Biocat), 10 nM Prostaglandin E2 (Santa Cruz Biotechnology), 10 μM Y-27632 (Selleck chemicals) and 100 mg/ml Primocin (Invivogen). After isolation, cells were kept in 4 conditions including medium as described (ENA), or supplemented with additional 3 uM SB202190 (Biomol) (ENAS), 50\% Wnt-conditioned medium and 20\% R-Spondin conditioned medium (WENRA) or both (WENRAS), as described by Fujii et al. \cite{Fujii2016ATumorigenesi  }. 
The tumor niche was determined after 14 days and organoids were subsequently cultured in the condition with best visible growth. 
Organoids were passaged every 7 days and medium was changed every 2-3 days. 25 PDO lines were established, data of all PDO lines including niche and growth rate are denoted in Table S1.

\subsection{Mouse Organoid Culture}
A heterozygous LSL-Kras G12D (B6.129S4-Krastm4Tyj/J) female mouse was crossed with a homozygous Rosa26-CreERT2 (B6.129-Gt(ROSA)26Sortm1(cre/ERT2)Tyj/J) male to generate offspring with a Tamoxifen activatable KRAS G12D allele. A single healthy LSL-Kras G12D CreERT2 mouse (male, 8 weeks) was sacrificed for organoid generation. 

Mouse colon organoids were isolated based on work by Sato et al \cite{Sato2009}. After cervical dislocation of the sacrificed mouse the colon was prepared and excised between caecum and rectum. The tissue was stored on ice in cooled DPBS (Life technologies), cut open lengthwise and washed three times with DPBS. After thorough washing, colon fragments were cut into 2mm pieces and incubated in a 5mM EDTA/DPBS (Sigma) solution for 60 minutes on a rocking table. Digested fragments were allowed to settle and resuspended in DMEM (Life technologies) by repeated up- and down-pipetting with a serological pipette. The resulting crypt suspension was filtered with a 70ul filter (Falcon), crypts were counted and centrifuged at 150g, 10min, 4C. The resulting pellet was resuspended in 10mg/ml Matrigel (Corning) and plated on prewarmed 6-well suspension plates (Greiner). After 30-60 minutes of solidification, droplets were overlaid with complete organoid growth medium and incubated at 37C, 5\% CO2 in atmospheric air.

After isolation, colon organoids were cultured in solidified BME R1 (10mg/ml) droplets and overlaid with genotype and experiment dependent growth medium. The medium was exchanged every 48-72 h. Complete colon organoid medium, termed WENRAS, contained 30\% advanced DMEM/F12 (Life Technologies) supplemented with 1\% v/v penicillin/streptomycin solution (Life Technologies), 1\% v/v HEPES buffer (Life Technologies) and 1\% v/v Glutamax (Life Technologies), 50\% Wnt3A conditioned medium, 20\% R-spondin1-FC conditioned medium. 
The medium was further supplemented with recombinant Noggin (100 ng/ml), 1x B27 (1x), n-Acetyl-cysteine (1.25 mM), Nicotinamide (10 mM), EGF (50 ng/ml), 500 nM A83-01 (Tocris), SB202190 (3 μM), Y-27632 (10 µM) and Primocin (100 µg/ml). All small molecule inhibitors were dissolved in DMSO. 
APC mutant colon organoid lines were cultured without Wnt and R-spondin conditioned medium, which was replaced by basal medium instead.
Organoids were passaged weekly by digestion with TrypLE (Gibco) and resuspension in BME R1 (10mg/ml). 
Organoids were regularly tested for Mycoplasma contaminations.  

\subsection{Genetic editing of organoids}
An sgRNA targeting the murine ortholog of the APC mutation cluster region (MCR) was designed using E-CRISP(Heigwer et al., 2014). The Apc targeting sgRNA was cloned into the one-vector plasmid pSpCas9(BB)-2A-Puro (PX459) V2.0 according to Ran et al. \cite{Ran2013}. Briefly, the vector was digested with Bbs1-HF (Thermo) and the phosphorylated and annealed oligonucleotide sgApc1 (sgAPC1 F and -R) was ligated using T4-Ligase (Thermo). The construct was transformed into chemically competent bacterial cells (Stellar, Clontech) and plated on Carbenicillin agar. Individual colonies were isolated and sequencing of plasmid DNA from cultured colonies confirmed successful molecular cloning.   
Extracted organoids (termed “wildtype”, “WT”) were cultured for multiple passages before transfection of the plasmid with Lipofectamine 2000 (Thermo). Here, grown organoids were digested with TrypLE (Gibco) and treated with Lipofectamine and plasmid DNA according to the manufacturer’s protocol. Transfected organoids were seeded in BME R1 (10mg/ml) and Wnt3A/R-Spondin1-Fc withdrawal was started 7 days after transfection. Surviving organoids were cultured continuously without Wnt3A and R-Spondin1-Fc conditioned medium (termed “A”).
To activate oncogenic Kras, Wildtype and APC mutant organoid lines were treated for 7 days with 0.5uM 4-Hydroxytamoxifen (Sigma) without EGF in the medium. 4-Hydroxytamoxifen was dissolved in Ethanol. After treatment, organoids were cultured with EGF containing media thereafter (termed “K” or “AK”, respectively).

\section{Biochemical assays}

%% stopped citing

\subsection{Amplicon Sequencing of Patient derived organoids}
DNA was isolated from 19 organoid cultures with the DNA blood and tissue kit (Qiagen). Sequencing libraries were prepared with a custom panel (Tru-Seq custom library kit, Illumina) according to the manufacturers protocol and sequenced on a MiSeq (Illumina). Targeted regions included the most commonly mutated hot spots in colorectal cancer in 46 genes captured with 157 amplicons of approximately 250bp length. A list of targeted hot- spots that were sequenced can be found in supplementary table S5. After mapping of the reads to GRC38 reference genome using Burrows-Wheeler Aligner (BWA), data were analyzed using the Genome Analysis Toolkit (GATK) (McKenna et al., 2010; Van der Auwera et al., 2013). Base recalibration was performed and variants were called using MuTect2 pipeline. Variants with a variant frequency below 10\%, with less than 10 reads, or with a high strand bias (FS<60) were filtered out. Variants were annotated with Ensemble variant effect predictor (McLaren et al., 2016) and manually checked and curated using integrative genomics viewer, if necessary \cite{Thorvaldsdottir}. Only non-synonymous variants present in COSMIC (Forbes et al., 2015) were considered true somatic cancer mutations.
Also, all variants annotated “benign” according to PolyPhen database and “tolerated” in SIFT database were excluded, as well as variants with a high frequency in the general population as determined by a GnomAD (Lek et al., 2016) frequency of >0.001.

\subsection{Amplicon Sequencing of Mouse Organoids}
Amplicon sequencing was performed to validate the genetic perturbation of Apc. DNA from Apc targeted and untargeted organoid lines was prepared using the DNA Blood and Tissue Kit (Qiagen), according to the manufacturer’s tissue protocol including an RNAse digestion. The targeted region was PCR amplified using primers F1 to R2, and sequencing libraries were prepared according to the manufacturer’s protocol. Libraries were sequenced on a MySeq (Illumina) using 100bp single end reads. 

\subsection{Genomic PCR of KRAS G12D allele}
To confirm activation of oncogenic Kras in 4-Hydroxytamoxifen treated lines, genomic DNA was isolated from all 4 organoid lines as described above. Presence or absence of the Lox-STOP-Lox cassette was evaluated by PCR according to the Kras G12D conditional PCR protocol by Tyler Jacks’ group(Jackson et al., 2001). Briefly, primers #2 and #3 were used for genotyping on genomic DNA using the Q5 PCR protocol (NEB).

\subsection{Western Blot}
Organoids were cultured in Matrigel (Corning). Organoids were collected, and cells were isolated using Matrisperse (Corning) for 40 minutes on a rocking table. Isolated organoids were lysed in RIPA buffer (Sigma) with Protease inhibitor (Sigma) and Phosphatase inhibitor 3 (Sigma). Protein concentration was measured using the Pierce BCA kit (Thermo) according to the manufacturers protocol and samples were loaded onto NuPage gels (Thermo). Western Blotting was performed with following antibodies: anti-p(hospho)-Erk (1:2000), anti-Erk (1:1000), anti-SREBP1 (1:1000), anti-SCAP (1:5000) and anti-beta-actin-HRP (1:150,000).

\subsection{Organoid Growth Patterns}
Organoids were passaged and seeded in 4 different growth media with medium changes every 48h. Images were taken 120h after seeding.  

\subsection{RT-qPCR}
Organoids were passaged and seeded in 4 different growth media with medium changes every 48h. After 120h, organoid RNA was isolated using the RNAEasy Kit (Qiagen) with beta-Mercaptoethanol (Invitrogen) and a DNAse digestion step. cDNA was synthesized using Oligo-dT primers (Thermo), RiboLock Ribonuclease inhibitor (Thermo) and Revert Aid H Minus reverse transcriptase (Thermo). RT-qPCR was performed using the ROCHE UPL kit (Roche), Sdha and Hprt expression levels were used as controls and averaged. Relative transcript abundance was measured using the ddCT method. RT-qPCR primers targeting Axin2, Ccnd, as well as the mouse reference genes Sdha and Hprt were used. 

\subsection{Mass Spectrometry}
Organoids were cultured according to the screening protocol (above). Organoids were isolated with Matrisperse (Corning) as described above. Isolated organoids were lysed in Ammonium Bicarbonate lysis buffer (50mM, pH 8.2) with 2.5\% w/v SDC and 25U/ml Benzonase. 
Samples were applied to 1D-SDS-PAGE and fractionated. Gel pieces were extracted, cysteins residues reduced by DTT and carbamidomethylated using iodoacetamide. The samples were digested with Trypsin overnight.
Resulting peptides were loaded on a cartridge trap column, packed with Acclaim PepMap300 C18, 5µm, 300Å wide pore (Thermo) and segregated in a 60 min gradient from 3\% to 40\% ACN on a nanoEase MZ Peptide analytical column (300Å, 1.7 µm, 75 µm x 200 mm, Waters). Eluted peptides were analyzed by an online coupled Q-Exactive-HF-X mass spectrometer.

\subsection{Expression Profiling}
Organoid RNA was isolated from 19 PDO lines with the RNeasy mini kit after snap freezing organoids on dry ice. Samples were hybridized on Affymetrix U133 plus 2.0 arrays. Raw microarray data were normalized using the robust multi-array average (RMA) method (Irizarry et al., 2003) followed by quantile normalization as implemented in the ‘affy’ (Gautier et al., 2004) R/Bioconductor (Huber et al., 2015) package. In order to exclude the presence of batch effects in the data, principal component analysis and hierarchical clustering were applied. Consensus molecular subtypes were determined as described previously (Guinney et al., 2015) using the single sample CMS classification algorithm with default parameters as implemented in the R package ‘CMSclassifier’. In all cases, differential gene expression analyses were performed using a moderated t-test as implemented in the R/Bioconductor package ‘limma’ (Ritchie et al., 2015). Gene set enrichment analyses were performed using ConsensusPathDB (Kamburov et al., 2013) for discrete gene sets or GSEA as implemented in the ‘fgsea’ (Sergushichev, 2016; Subramanian et al., 2005) R/Bioconductor package for ranked gene lists.

Mouse organoids were cultured according to a standardized screening protocol. Briefly, organoid models were seeded and cultured for 72h in WENRAS+Y and additional 96h in ENR-Y. Samples were harvested after 7 days and RNA was isolated using the RNAEasy Kit (Qiagen) as described above. Transcript expression levels were measured using MoGene-2\_0-st chips (Affymetrix).    

\section{Organoid profiling}

\subsection{Cell Seeding during Compound testing}
PDO drug profiling followed a standardized protocol with comprehensive documentation of all procedures. Organoids were collected and digested in TrypLE Express (Life technologies). Fragments were collected in basal medium with 300 U/ml DNAse and strained through a 40μm filter to achieve a homogeneous cell suspension with single cells and small clusters of cells, but without large organoid fragments. 384 well μclear assay plates (Greiner) were coated with 10μL BME V2 (Trevigen) at a concentration of 6.3 mg/ml in basal medium, centrifuged and incubated for >20 min at 37° C to allow solidification of the gel. PDO cell
clusters together with culture medium (ENA) and 0,8 mg/ml BME V2 were added in a volume of 50μl per well using a Multidrop dispenser (Thermo Fisher Scientific). Plates were sealed with a plate-loc (Agilent) and centrifuged for additional 20 min allowing cells to settle on the pre-dispensed gel. Cell number was normalized before seeding by measuring ATP levels in a 1:2 dilution series of digested organoids with CellTiter-Glo (Promega). The number of cells matching 10,000 photons was seeded in each well. After seeding of organoid fragments, plates were incubated for three days at 37°C to allow organoid formation before addition of compounds. Two biological replicates of each PDO line from different passages were profiled at different time points. In total, 16 PDO lines underwent compound profiling.

Mouse organoid screening was performed as described above with slight modifications. Clotting of organoid fragments was avoided by adding 10 U/ml of bovine DNAse1 to the medium during filtration. The cell viability of digested fragment suspensions was estimated using Cell-Titer-Glo (Promega). 40ul of cell suspension was mixed with 40ul of undiluted reagent and measured after 30 minutes on a Mithras plate reader (Berthold). Cell fragments with a viability corresponding to 5000 photons were seeded per well on pre-coated 384 well plates using a Multidrop peristaltic pump robot (Thermo). After 72 hours of organoid expansion in WENRAS+Y, the medium was changed to ENR-Y and compound libraries were added using a BiomekFX (Beckmann). Screening plates were further incubated for 96h before a cell permeability dye Image-IT DeadGreen was added for 4 hours and cells were fixed in a 3\% PFA buffer supplemented with 1\% BSA. After fixation, plates were stored for up to 4 days or directly stained with DAPI (Sigma) and Phalloidin-TRITC. Processed plates were imaged using an Incell6000 automated line-scanning confocal fluorescence microscope.

\subsection{Compound Libraries}
Two compound libraries were used for screening: A library containing 63 clinically relevant drugs (clinical cancer library) and a large library of 464 compounds targeting kinases and stem cell or developmental pathways associated genes (KiStem library). The clinical cancer library was manually curated by relevance for current (colorectal) cancer therapy, mechanism of action and potential clinical applicability. Compounds of this library are in clinical use or at least in phase I/II clinical trials. Five concentrations per compound were screened (five-fold dilutions). The concentrations were determined by analysis of literature data from previous 3D and 2D drug screens and own experiments. A list of compounds included in this library and maximum concentrations used can be found in S3. The KiStem library includes 464 compounds targeting a diverse set of kinases and stem cell relevant pathways S4. All compounds in this library were used in a concentration of 7.5μM. All compounds were obtained from Selleck chemicals. Compounds of both libraries were arranged in an optimized random layout. We stored compound libraries in DMSO at -80 C.

\subsection{Compound Treatment}
Medium was aspirated from all screening plates and replaced with fresh ENA medium devoid of Y-27632, resulting in 45μl volume per well. Drug libraries were diluted in basal medium and subsequently 5μl of each compound was distributed to screening plates. 
All liquid handling steps were performed using a Biomek FX robotic system (Beckmann Coulter). Plates were sealed and incubated with the compounds for four days. All PDO lines underwent profiling with the clinical cancer library, while the KiStem library was used with 13 PDO lines.

\subsection{Luminescence Viability Read Out}
Plates undergoing viability screening were treated with 30μl CellTiter-Glo reagent after medium aspiration with a Biomek FX. After incubation for 30 minutes, luminescence levels were measured with a Mithras reader (Berthold technologies).

For mouse experiments, before compound addition, organoid viability was measured using Cell-Titer-Glo (Promega) and WENR-Y was added to the remaining plates. Cell viability was measured after compound exposure as described above. The pre-treatment viability of organoids was used to estimate growth-rate controlled dose-response curves according to Clark et al.(Hafner et al., 2016) Measurements of GR metrics for was not robust for slow proliferating WT lines. Therefore, we omitted plotting dose-response curves for WT lines when a GRfit was conducted. Dose response curves of relative viability, including the WT line can be found in the supplementary figures. 

\subsection{Image-based Phenotyping}
Image-IT DeadGreen (Thermo Fisher) was added to the cultures with a Multidrop dispenser (Thermo Fisher) in 100nM final concentration and incubated for four hours. Afterwards, medium was removed and organoid cultures were fixed with 3\% PFA in PBS with 1\% BSA. Fixed plates were stored at 4° C for up to three days before permeabilization and staining. On the day of imaging, organoids were permeabilized with 0.3\% Triton-X-100 and 0.05\% Tween in PBS with 1\% BSA and stained with 0.1μg/ml TRITC-Phalloidin (Sigma) and 2μg/ml DAPI (Sigma). All liquid handling steps were performed with a BiomekFX. Screening plates were imaged with an Incell Analyzer 6000 (GE Healthcare) line-scanning confocal fluorescent microscope. We acquired 4 fields per well with z-stacks of 16 slices at 10x magnification. The z-steps between the 16 slices had a distance of 5μm, the depth of field of each slice was 3.9μm.

\subsection{Immunohistochemistry}
PDOs were fixed for 20 min in 4\% (v/v) Roti Histofix (Carl Roth) followed by embedding into MicroTissues 3D Petri Dish micromolds (Sigma Aldrich) using 2\% (w/v) Agarose LE (Sigma) in PBS supplemented with 0.5 mM DTT. Thereafter, PDOs were subjected to dehydration steps and embedding in paraffin. Formalin-Fixed Agarose/Paraffin-Embedded sections (3- 5μm) were manually cut from blocks with a microtome (Leica RM 2145) and transferred to glass slides (Superfrost, Thermofisher Scientific) before H&E staining using automated staining devices.

\subsection{Proliferation Assay}
Organoids were collected, digested, strained, normalized and seeded as in the drug profiling protocol described above. Medium change was performed on days 2 and 4. Viability of plated organoids was measured 2, 4 and 8 days after seeding using CellTiter-Glo as described above.

\section{Image analysis}

\subsection{Image Processing}
Microscopic image z-stacks were compressed to HDF5 format for archival and underwent maximum contrast projection using the R/Bioconductor package MaxContrastProjection for further processing of the images. Segmentation of the projections based on intensity did not sufficiently identify organoids. Instead, we trained a deep convolutional neural network (DCNN) on the partially correct intensity segmentation, leveraging the robustness of DCNNs with regards to mislabeled training data and eliminating the need for expensive manual annotations. Standard image features, including shape, moment, intensity, and Haralick texture features on multiple scales, were extracted using the R/Bioconductor package EBImage (Pau et al., 2010). Of note, the strong diversity of unperturbed organoid phenotypes between PDO lines did not allow the definition of a core set of individual reproducible descriptive features across all screened organoids. Therefore, no correlation-based filtering of features was done, allowing comparisons between different PDO lines. Out-of-focus objects were programmatically removed from the dataset using a feature based random forest classifier. Two PDO lines (D015T01 and D021T01) had to be excluded from analysis because of too many of out-of-focus elements.

\subsection{Drug-Induced Phenotypes}
A principal component analysis (PCA) was performed on the entire dataset to reduce the dimensionality. 25 principal components were selected, explaining approx. 81\% of the total variance within the dataset. A linear support vector machine (SVM) was trained per line and treatment (and per concentration where applicable) to differentiate treated organoids from negative controls based on the PCA-transformed features (Loo et al., 2007). To allow comparison between various PDO lines and drug perturbations, the distributions of features describing organoids from different batches were adjusted. Drugs were categorized as either active or inactive based on the accuracy of the SVM. The histogram of accuracies made a threshold of 85\% the most intuitive. The direction of the vector perpendicular to the SVM hyperplane was interpreted as the drug-induced effect. Drugs were clustered with regard to the angles of their corresponding effect vectors in PCA-feature space.

\subsection{Live-Dead Classification}
A random forest classifier was trained on the original single organoid features to differentiate living from dead organoids. Organoids treated with DMSO were used as negative (i.e. living) controls while organoids treated with Bortezomib and SN-38 at the two highest concentrations were used as positive (i.e. dead) controls. Visual inspection of the projected images confirmed our choice of positive controls. Binary classification results were averaged within wells to obtain viability scores ranging from 0 to 1, indicating how lethal a treatment was. A separate classifier was trained for each individual line to ensure inter-line independence.

\subsection{Analysis of Luminescence Data and Dose-Response Relationships}
Raw luminescence data of each plate were first normalized using the Loess-fit method
(Mpindi et al., 2015) in order to correct for edge effects where increased luminescence intensity was observed along the edges of each plate. Subsequently, each plate was normalized by division with the median luminescence intensity of the DMSO controls. Drug response Hill curves (DRC) were fitted and area under the curve values were calculated for each DRC using the ‘PharmacoGx’ (Smirnov et al., 2016) R/Bioconductor package.

\end{flushleft}